libraries/DataFlash/LogStructure.h

```diff
@@ -480,6 +480,8 @@ struct PACKED log_Current {
     float    battery_voltage;
     float    current_amps;
     float    current_total;
+    int16_t  temperature; // degrees C * 100
```

Comment:
Okay the comment might actually be more misleading... We're setting the FMT_LABEL to have the unit * 100 already, so this will result in your graping program automatically dividing this by 100 again for you. The reason I selected this resolution is that our only current source of temperature information is only good to 0.1 C, so this will convey that level of accuracy, and I didn't want to use a 4 byte storage like a float. Because you will automatically see this divided again I think this is the correct approach.

Does that make sense?